docs/topics/forms/modelforms.txt

```diff
@@ -972,6 +967,30 @@ so long as the total number of forms does not exceed ``max_num``::
 A ``max_num`` value of ``None`` (the default) puts a high limit on the number
 of forms displayed (1000). In practice this is equivalent to no limit.
 
+Also, ``extra=0`` doesn't prevent creation of new model instances as you can
+:ref:`add additional forms with JavaScript <understanding-the-managementform>`
+or send additional POST data.
+
+Edit-only mode for the model formsets
+-------------------------------------
+
+.. versionadded:: 4.0
+
+.. attribute:: modelformset_factory.edit_only
```

Comment:
Hello, @felixxm. I have trouble with test for `formset_factory`. I added `edit_only` kwarg to this function just to pass it to the `modelformset_factory`, as Jacob [suggested](https://github.com/django/django/pull/14725#discussion_r705809527). In `formset_factory` this kwarg don't do any actions and I, actually, don't know how to test it. Can you advise me something?

Reply:
@felixxm. As I said in previous comment, `edit_only` kwarg in `formset_factory` don't change behavior of this function. Do we need move this docs to the `docs/topics/forms/formsets.txt`? I suggest leave this section and add new to the `docs/topics/forms/formsets.txt`. I would be grateful for your help.

Reply:
Hi, @jacobtylerwalls! I would appreciate if you can help me with the questions above.

Reply:
Hi @Nakmak98, hope I can help out here. Sounds like we need a test for `formset_factory`. Maybe you can try this?

```python
        form = modelform_factory(Author, fields='__all__')
        AuthorFormSet = formset_factory(form, BaseModelFormSet, edit_only=True)
        AuthorFormSet.model = Author
```

Essentially we are just repeating the same steps that `modelformset_factory()` performs without calling it. I added this to the end of your test and repeated the assertions and it passed. Maybe there's a way to factor it so that you don't have to repeat identical assertions. Let me know if this helps. 👍🏻